File: app/src/main/java/com/softaai/designpatterns/java/creational/builder/meal/item/ColdDrink/Coke.java
package com.softaai.designpatterns.java.creational.builder.meal.item.ColdDrink;

/**
 * Created by Amol Pawar on 01-04-2019.
 * softAai Apps
 */
public class Coke extends ColdDrink {
    @Override
    public String name() {
        return "Coke";
    }

    @Override
    public float price() {
        return 30.0f;
    }
}

File: app/src/main/java/com/softaai/designpatterns/kotlin/creational/builder/meal/item/ColdDrink/Coke.kt
package com.softaai.designpatterns.kotlin.creational.builder.meal.item.ColdDrink


/**
 * Created by Amol Pawar on 01-04-2019.
 * softAai Apps
 */
class Coke : ColdDrink() {
    override fun price(): Float {
       return 30.0f
    }

    override fun name(): String {
        return "Coke"
    }
}